Client/src/main/java/com/minecrafttas/tasbattle/mixin/tickratechanger/MixinTickrateChangerServer.java

```diff
@@ -0,0 +1,24 @@
+package com.minecrafttas.tasbattle.mixin.tickratechanger;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+
+import com.minecrafttas.tasbattle.TASBattle;
+
+import net.minecraft.server.MinecraftServer;
+
+/**
+ * Changes the tickrate of the server
+ * @author Pancake
+ *
+ */
+@Mixin(MinecraftServer.class)
+public class MixinTickrateChangerServer {
+
+	@ModifyConstant(method = "runServer", constant = @Constant(longValue = 50L))
+	public long runServer(long out) {
+		return TASBattle.tickratechanger.getMsPerTick();
+	}
+	
+}
```

Comment:
won't be used

Reply:
I thought about testing some things in singleplayer before trying to test it on the server... Therefore I added that from the original...

Reply:
okay